File: C/0005/main.c
#include <stdio.h>

int main()
{
    
    int count = 0;
    int i = 2;
    int m, n;
    int sum = 0;
    scanf ( "%d %d", &n, &m );

    while ( count < m )
    {
        int flag = 1;
        for ( int j = 2; j < i; j++ )
        {
            if ( i % j == 0 )
            {
                flag = 0;
                break;
            }
        }
        if ( flag == 1 )
        {
            count++;
            if ( count >= n && count <= m )
            {
                sum += i;
            }
        }
        i++;
    }
    printf ( "%d", sum );

    return 0;
}

File: Python/0005/main.py
# -*- coding: utf-8 -*-
"""
Created on Tue Dec  9 11:10:56 2014

@author: MasterMac
"""
import math

n = int(raw_input())
sum = 0
count = 2 

while count < n:
    for i in range(2, int(math.sqrt(count)) + 1):
        if count % i == 0:
            break
    else:
        sum += count 
    count += 1

print sum
